pkgs/by-name/ut/ut1999/package.nix

```
@@ -91,23 +92,32 @@ stdenv.mkDerivation {
       autoPatchelfHook
     ]
     ++ lib.optionals stdenv.hostPlatform.isDarwin [
+      makeWrapper
       undmg
     ];
 
   installPhase =
     let
       outPrefix =
-        if stdenv.hostPlatform.isDarwin then "$out/UnrealTournament.app/Contents/MacOS" else "$out";
+        if stdenv.hostPlatform.isDarwin then
+          "$out/Applications/UnrealTournament.app/Contents/MacOS"
+        else
+          "$out";
     in
     ''
       runHook preInstall
-      mkdir -p $out
+      mkdir -p $out/bin
     ''
     + lib.optionalString (stdenv.hostPlatform.isLinux) ''
-      mkdir -p $out/bin
+      cp -r "./*" $out
```

I think the quoting got a little messed up.

```suggestion
      cp -r ./* $out
```

---

Oh yeah, moving it out of the nix expression, it should have lost the quotes. Thanks!

---

jup, that was it, I've added the fix.